### PR TITLE
[CLEANUP] Remove deprecated images and move stuff around a bit.

### PR DESCRIPTION
**Pull request by cafuego**

### CODE REVIEW

File: alpine-php/builder/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-include ../../../Makefile.conf
 include ../../Makefile.conf
+include ../Makefile.conf
```
Comment (emmajane) [inline alpine-php/builder/Makefile:2]: Technically the one in one directory up is only `Makefile` not `Makefile.conf` does that Make a difference?

File: alpine-php/php7-k8s/Makefile [hunk 2]
```diff
@@ -3,7 +3,7 @@ include ../Makefile.conf
 
 UPSTREAM=$$(UPSTREAM)
 VERSION=$$(VERSION)
-IMAGE=base-php5
```
Comment (emmajane) [inline alpine-php/php7-k8s/Makefile:6]: so long, sucker!

**Review by emmajane — APPROVED** [review]
i have questions about the missing `conf`

File: alpine-php/builder/Makefile (hunk 1 above)
Comment (cafuego) [reply alpine-php/builder/Makefile:2]: Wut? Not on my source tree it's not.

Comment (emmajane) [reply alpine-php/builder/Makefile:2]: ummm. I button mashed and was just looking in the UI. :( unno.

Screenshot 2020-11-18 at 08 06 48
Screenshot 2020-11-18 at 08 07 18
Screenshot 2020-11-18 at 08 07 07

Comment (emmajane) [reply alpine-php/builder/Makefile:2]: (And then if I go up one more in the directory tree it's `Makefile.conf`)

Screenshot 2020-11-18 at 08 10 21

Comment (cafuego) [reply alpine-php/builder/Makefile:2]:
```
cafuego@calvados:~/Development/unocha/docker-images/alpine-php/builder (cafuego/cleanup-deprecated-images $%)
; head Makefile 
include ../../Makefile.conf
include ../Makefile.conf
cafuego@calvados:~/Development/unocha/docker-images/alpine-php/builder (cafuego/cleanup-deprecated-images $%)
; ls -la ../Makefile.conf ../../Makefile.conf 
-rw-r--r--  1 cafuego  staff  1664 18 Sep 15:36 ../../Makefile.conf
-rw-r--r--  1 cafuego  staff    10 17 Aug  2018 ../Makefile.conf
```

All as it should be.

**Pull request merged**